packages/generator-single-spa/src/common-templates/typescript/declarations.d.ts.ejs

```diff
@@ -0,0 +1,47 @@
+declare module "*.html" {
+  const rawHtmlFile: string;
+  export = rawHtmlFile;
+}
+
+declare module "*.bmp" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.gif" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.jpg" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.jpeg" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.png" {
+  const src: string;
+  export default src;
+}
+
+declare module "*.webp" {
+  const src: string;
+  export default src;
+}
+
+<% if (react) { $>
```

Comment:
svgs should be supported in non-react projects. Either through an `<% else %>` or by removing the (possibly unnecessary) react references inside of the svg.